recipes/yattag/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "yattag" %}
+{% set version = "1.11.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 94210945c868f696a31812a510c01724d61c9a4d279eb4adf340c0d2a4c0bfd7
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
```suggestion
  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Reply:
I see some recipes specify python and some specify {{ PYTHON }}.  Is there some edge case where one or the other won't work?  Also, is there a reason to leave out --ignore-installed here?